distributionviewer/core/static/js/app/components/views/filtered-charts.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export class FilteredCharts extends React.Component {
+  constructor(props) {
+    super(props);
+    this.filteredClass = ' filtered';
+  }
+
+  componentDidMount() {
+    var wrapper = document.getElementsByClassName('global-wrapper')[0];
+    wrapper.className += this.filteredClass;
```

Comment:
Good catch. Sounds good to me as long as we don't need to support IE < 11.

http://caniuse.com/#search=classlist